runtime/Cargo.toml

```diff
@@ -15,7 +15,7 @@ serde = { version = "1.0.217", features = ["derive"] }
 serde_json = "1.0.134"
 keyring = { version = "3.6.1", features = ["apple-native", "windows-native", "sync-secret-service"] }
 arboard = "3.4.1"
-tokio = { version = "1.42", features = ["rt", "rt-multi-thread", "macros"] }
+tokio = { version = "1.42", features = ["rt", "rt-multi-thread", "macros", "process"] }
```

Comment:
Why do we now need the `process` feature?

Reply:
Okay, I just figured it out myself: We need this to start external processes. Depending on the data type, we call `pandoc`.